extension/background.js

```diff
@@ -1,82 +1,82 @@
-    new ScriptExecution(tab.id)
-        .executeScripts(
-            "third_party/lodash-4.17.15.js",
-            "third_party/jquery-3.5.1.min.js",
-            "third_party/jquery.mark-8.11.1.min.js",
-            "third_party/jquery-ui.1.12.1.min.js",
-            "third_party/readability-1.7.1.js",
-            "third_party/tokenize-text.js",
-            "third_party/tokenize-english.js",
-            "lib/settings_wrapper.js",
-            "lib/document_parser.js",
-            "lib/tracker.js",
-            "lib/time_tracker_view.js",
-            "lib/doc.js",
-            "lib/settings_view.js",
-            "content_script.js"
-        )
-        .then(s => s.injectCss("content.css"))
-        .then(s => {
-            // Send a message which will get handled by content_script.js
-            // This ensures that the same isolated world which has the first and only 
-            // script execution can handle the event.
-            // The alternative approach of executeScripts and rerunning content_script.js
-            // does not work because it will execute in a different isolated world,
-            // and will have lost access to the previous javascript vars and event handlers.
-            // See https://stackoverflow.com/a/8916706/4143394
-            chrome.tabs.sendMessage(tab.id, {command: "toggleUI"}, function(response) {});
-        });
-});
+//     // All the scripts below have IFNDEF protection patterns to ensure
+//     // they only run once per page. 
+//     new ScriptExecution(tab.id)
+//         .executeScripts(
+//             "third_party/lodash-4.17.15.js",
+//             "third_party/jquery-3.5.1.min.js",
+//             "third_party/jquery.mark-8.11.1.min.js",
+//             "third_party/jquery-ui.1.12.1.min.js",
+//             "third_party/readability-1.7.1.js",
+//             "third_party/tokenize-text.js",
+//             "third_party/tokenize-english.js",
+//             "lib/settings_wrapper.js",
+//             "lib/document_parser.js",
+//             "lib/tracker.js",
+//             "lib/time_tracker_view.js",
+//             "lib/doc.js",
+//             "lib/settings_view.js",
+//             "content_script.js"
+//         )
+//         .then(s => s.injectCss("content.css"))
+//         .then(s => {
+//             // Send a message which will get handled by content_script.js
+//             // This ensures that the same isolated world which has the first and only 
+//             // script execution can handle the event.
+//             // The alternative approach of executeScripts and rerunning content_script.js
+//             // does not work because it will execute in a different isolated world,
+//             // and will have lost access to the previous javascript vars and event handlers.
+//             // See https://stackoverflow.com/a/8916706/4143394
+//             chrome.tabs.sendMessage(tab.id, {command: "toggleUI"}, function(response) {});
+//         });
+// });
```

Comment:
Left in case we want to turn it back; I can also just delete this file

Reply:
I would just delete it. One less file + package.json line.